### ISSUE TITLE
Build error: uninitialized in this function

### ISSUE THREAD

**Issue by daissi** [statgen/libStatGen#19]
Hi,
libStatGen fails to build [1-2] in Debian with this error:

```
g++  -O4 -fPIC -pipe -Wall -Wdate-time -D_FORTIFY_SOURCE=2 -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -Wl,-z,relro -Wl,-z,now -Werror -Wno-format-truncation  -Wno-strict-overflow  -I../include -I.   -D__ZLIB_AVAILABLE__ -D_FILE_OFFSET_BITS=64 -D__STDC_LIMIT_MACROS  -o obj/CigarRoller.o -c CigarRoller.cpp -DVERSION="\"1.0.0\""
Chromosome.cpp: In constructor 'Chromosome::Chromosome(const string&, unsigned int, bool)':
Chromosome.cpp:42:15: error: '*<unknown>.Chromosome::gs' is used uninitialized in this function [-Werror=uninitialized]
     if (this->gs) delete gs;
         ~~~~~~^~
```

Best,
Dylan

[1] https://buildd.debian.org/status/package.php?p=libstatgen
[2] https://bugs.debian.org/905368

**Comment by jonathonl** [comment]
NOTE:
I don't think this class is being used internally. Otherwise, it would have crashed a long time ago. These delete calls in the constructor should be removed, or the class should be purged all together.

**Comment by daissi** [comment]
Hi @jonathonl ,
Any news of this bug?
It prevents the migration of libstatgen and minimac4 to Debian/testing and Ubuntu.

**Comment by jonathonl** [comment]
Fixed with 3a10afe08c209dbacc3eca7794a5a8c0a42ee566.
